numba/np/ufunc/gufunc.py

```diff
@@ -102,6 +102,30 @@ def types(self):
     def identity(self):
         return self.ufunc.identity
 
+    @property
+    def signature(self):
+        return self.ufunc.signature
+
+    @property
+    def accumulate(self):
+        return self.ufunc.accumulate
+
+    @property
+    def at(self):
+        return self.ufunc.at
+
+    @property
+    def outer(self):
+        return self.ufunc.outer
+
+    @property
+    def reduce(self):
+        return self.ufunc.reduce
+
+    @property
+    def reduceat(self):
+        return self.ufunc.reduceat
```

Comment:
I think it'd be a good idea to add tests for these properties such that future refactors don't reintroduce this bug or similar.